__init__.py

```diff
@@ -122,6 +123,13 @@ def _git_push(self):
     def _get_data_location(self):
         return self.data_location
 
+    def _get_breaches(self, entry, last_modified):
```

Comment:
It's named _get_breaches but actually returns just if this single entry was found to be in a breach? Could you rename this function, something like `_get_breach_info` or so?

Reply:
done!